homeassistant/components/telegram_bot/broadcast.py

```diff
@@ -1,17 +1,6 @@
 """Support for Telegram bot to send messages only."""
-import logging
 
-from . import initialize_bot
 
-_LOGGER = logging.getLogger(__name__)
-
-
-async def async_setup_platform(hass, config):
+async def async_setup_platform(hass, bot, config):
     """Set up the Telegram broadcast platform."""
-    bot = initialize_bot(config)
```

Comment:
This is no longer necessary because it's done before calling `async_setup_platform` and is passed to `async_setup_platform`: https://github.com/home-assistant/core/blob/dev/homeassistant/components/telegram_bot/__init__.py#L322-L329